src/commands/agent.rs

```diff
@@ -91,7 +97,10 @@ fn agent_status_handler(url: &str, batch_list: &BatchList) -> Result<(), CliErro
             .status
             .as_ref()
         {
-            "COMMITTED" => break Ok(()),
+            "COMMITTED" => {
+                println!("Agent {} has been {}d", public_key, action);
```

Comment:
having the `d` works for now with both `create` and `authorize ` ending in an e but feels like a potential bug down the line - what about this?

```suggestion
                println!("{} action was successful on behalf of agent", action, public_key);
```

Reply:
Edited your suggestion to include missing placeholder `{}` for public_key